holidays/locale/en_US/LC_MESSAGES/AW.po

```diff
@@ -0,0 +1,87 @@
+# Aruba holidays en_US localization.
+# Authors: PPsyrius <ppsyrius@ppsyrius.dev>, (c) 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Python Holidays 0.27\n"
+"POT-Creation-Date: 2023-06-09 23:12+0700\n"
+"PO-Revision-Date: \n"
+"Last-Translator: PPsyrius <ppsyrius@ppsyrius.dev>\n"
+"Language-Team: Python Holidays localization team\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 3.3.1\n"
+
+#. New Year's Day
+#: ./holidays/countries/aruba.py:53
+msgid "Aña Nobo"
+msgstr "New Year's Day"
+
+#. Betico Croes Day
+#: ./holidays/countries/aruba.py:61
+msgid "Dia di Betico"
+msgstr "Betico Day"
+
+#. Carnival Monday
+#: ./holidays/countries/aruba.py:72
+msgid "Dialuna despues di Carnaval Grandi"
+msgstr "Carnival Monday"
+
+#. Monday before Ash Wednesday
+#: ./holidays/countries/aruba.py:75
+msgid "Dialuna prome cu diaranson di shinish"
+msgstr "Monday before Ash Wednesday"
+
+#. National Anthem and Flag Day
+#: ./holidays/countries/aruba.py:84
+msgid "Dia di Himno y Bandera"
+msgstr "National Anthem and Flag Day"
+
+#. Good Friday
+#: ./holidays/countries/aruba.py:90
+msgid "Bierna Santo"
+msgstr "Good Friday"
+
+#. Easter Monday
+#: ./holidays/countries/aruba.py:96
+msgid "Di dos dia di Pasco di Resureccion"
+msgstr "Easter Monday"
+
+#. King's Day.
+#: ./holidays/countries/aruba.py:108
+msgid "Dia di Rey"
+msgstr "King's Day"
+
+#. King's Day.
+#: ./holidays/countries/aruba.py:112
+msgid "Aña di Rey"
+msgstr "King's Day"
+
+#. Queen's Day.
+#: ./holidays/countries/aruba.py:115
+msgid "Aña di La Reina"
+msgstr "Queen's Day"
+
+#. Labor Day
+#: ./holidays/countries/aruba.py:132
+msgid "Dia di Obrero"
+msgstr "Labor Day"
+
+#. Ascension Day
+#: ./holidays/countries/aruba.py:138
+msgid "Dia di Asuncion"
+msgstr "Ascension Day"
+
+#. Christmas Day
+#: ./holidays/countries/aruba.py:144
+msgid "Pasco di Nacemento"
+msgstr "Christmas Day"
+
+#. Boxing Day
+#: ./holidays/countries/aruba.py:150
+msgid "Di dos dia di Pasco di Nacemento"
+msgstr "Boxing Day"
```

Comment:
```suggestion
msgstr "Second Day of Christmas"
```
It's also closed to the original (as in the Netherlands). "Boxing Day" is typical for US/UK and related cultures.